clusterloader2/testing/load/experimental/overrides/disable_kubeproxy.yaml

```diff
@@ -0,0 +1 @@
+PROMETHEUS_SCRAPE_KUBE_PROXY: false
```

Comment:
NIT: Rename to include prometheus or scrape, as we're not disabling kube-proxy but scraping kube-proxy

Reply:
Also move it outside experimental directory, I think there is a dir with prometheus overrides

Reply:
Done